components/count-badge/count-badge.js

```diff
@@ -34,6 +36,22 @@ class CountBadge extends RtlMixin(LitElement) {
 				type: Boolean,
 				attribute: 'hide-zero'
 			},
+			/**
+			 * Optionally specify an icon string to appear with the badge.
+			 * @type {string}
+			 */
+			icon: {
+				type: String,
+				reflect: true
+			},
+			/**
+			 * Optionally specify a size for the height/width of the icon.
+			 * @type {string}
+			 */
+			iconSize: {
```

Comment:
This size fixing is required for d2l-card-footer-link to cap all the icons at 18*18px. I don't expect any consumers to use this property directly. Any thoughts on making this private and not exposing the functionality?